cpp/ps0007/cpp_ps0007_solution.cpp

```diff
@@ -51,3 +59,4 @@ float Area(char shape, float dimension[])
 	else
 		return 0.0f;
 }
+// good work 👍 7/10

```

Comment:
- [x] program is in the correct path
- [ ] no syntax errors while compilation & no bugs or warnings while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [x] used array & function concepts for the program
- [ ] good design for function using DRY concept
- [ ] main returns non-zero/zero number according to the case

you can check the [problem solution](https://onlinegdb.com/wybZ67yJX) for more